plugins/node/opentelemetry-instrumentation-nestjs-core/package.json

```
@@ -0,0 +1,78 @@
+{
+  "name": "@opentelemetry/instrumentation-nestjs-core",
+  "version": "0.23.0",
+  "description": "OpenTelemetry NestJS automatic instrumentation package.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "clean": "rimraf build/*",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "compile": "npm run version:update && tsc -p .",
+    "compile:watch": "tsc -w",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version",
+    "prepare": "npm run compile",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "version:update": "node ../../../scripts/version-update.js"
+  },
+  "keywords": [
+    "opentelemetry",
+    "nestjs",
+    "nodejs",
+    "tracing",
+    "profiling",
+    "instrumentation"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "peerDependencies": {
+    "@opentelemetry/api": "^1.0.1"
+  },
+  "devDependencies": {
+    "@nestjs/common": "^8.0.5",
+    "@nestjs/core": "^8.0.5",
+    "@nestjs/microservices": "^8.0.5",
+    "@nestjs/platform-express": "^8.0.5",
+    "@nestjs/websockets": "^8.0.5",
+    "@opentelemetry/api": "1.0.1",
+    "@opentelemetry/context-async-hooks": "0.24.0",
+    "@opentelemetry/node": "0.24.0",
+    "@opentelemetry/tracing": "0.24.0",
+    "@types/mocha": "7.0.2",
+    "@types/node": "14.17.2",
+    "@types/semver": "^7.3.8",
+    "@types/vinyl-fs": "^2.4.12",
+    "codecov": "3.8.2",
+    "cross-env": "7.0.3",
+    "gts": "3.1.0",
+    "mocha": "7.2.0",
+    "nyc": "15.1.0",
+    "reflect-metadata": "^0.1.13",
+    "rimraf": "3.0.2",
+    "rxjs": "^7.2.0",
+    "rxjs-compat": "^6.6.7",
+    "semver": "^7.3.5",
+    "ts-mocha": "8.0.0",
+    "typescript": "4.3.2"
```

```suggestion
    "typescript": "4.3.5"
```
Maybe there are other deps which could be updated

---

There were! Thanks!